packages/core/src/fs/project-reader.ts

```diff
@@ -43,7 +43,7 @@ export class ProjectReader {
     { mutate, tempDirName, ignorePatterns, incremental, incrementalFile, force }: StrykerOptions
   ) {
     this.mutatePatterns = mutate;
-    this.ignoreRules = [...ALWAYS_IGNORE, tempDirName, ...ignorePatterns];
+    this.ignoreRules = [...ALWAYS_IGNORE, tempDirName, incrementalFile, ...ignorePatterns];
```

Comment:
... And add them here.

Reply:
I added `htmlReporter.fileName` and `jsonReporter.fileName`.